bitcoin-core-dev-tech/2015-02/jeremy-allaire-circle.es.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Círculo 
+transcript_by: Bryan Bishop
+translation_by: Blue Moon
+categories: ['core-dev-tech']
+speakers: ['Jeremy Allaire', 'Sean Neville']
+aliases: ['/bitcoin-core-dev-tech/bitcoin-devcore-2015/jeremy-allaire-circle/']
```

Comment:
this alias is not needed. It was only needed in the original because we changed the directory structure.